makefiles/Makefile.devhub-content-new

```diff
@@ -0,0 +1,59 @@
+GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
+USER=$(shell whoami)
+
+STAGING_URL="http://docs-devhub-staging.s3-website-us-east-1.amazonaws.com"
+STAGING_BUCKET=docs-devhub-staging
+
+STRAPI_URL=http://54.219.137.111:1337
+
+PRODUCTION_BUCKET=developer-hub-prod
+PRODUCTION_URL=https://mongodb.com/developer
+
+PROJECT=devhub
+MUT_PREFIX ?= $(PROJECT)
+REPO_DIR=$(shell pwd)
+
+COMMIT_HASH=$(shell git rev-parse --short HEAD)
+
+GITHUB_USER = $(shell printenv GITHUB_BOT_USERNAME)
+GITHUB_PASS = $(shell printenv GITHUB_BOT_PASSWORD)
+SNOOTY_DB_USR = $(shell printenv MONGO_ATLAS_USERNAME)
+SNOOTY_DB_PWD = $(shell printenv MONGO_ATLAS_PASSWORD)
+
+include ~/shared.mk
+
+update-snooty:
+	cd ../../snooty-devhub; \
+	git fetch --all; \
+	git reset --hard origin/master; \
+	npm install --production
+
+get-build-dependencies:
+	@curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/devhub-content.yaml > ${REPO_DIR}/published-branches.yaml
+
+next-gen-html: update-snooty
+	# snooty parse and then build-front-end
+	@echo ${SNOOTY_DB_PWD} | snooty build "${REPO_DIR}" "mongodb+srv://${SNOOTY_DB_USR}:@cluster0-ylwlz.mongodb.net/snooty?retryWrites=true" --commit "${COMMIT_HASH}"; \
+	if [ $$? -eq 1 ]; then \
+		exit 1; \
+	else \
+		exit 0; \
+	fi
+	rsync -az --exclude '.git' ${REPO_DIR}/../../snooty-devhub ${REPO_DIR}; 
+	cp ${REPO_DIR}/.env.production ${REPO_DIR}/snooty-devhub; 
+	curl https://raw.githubusercontent.com/mongodb/devhub/master/gatsby-config.prod-new.js > ${REPO_DIR}/snooty/gatsby-config.js
```

Comment:
This is important, as in this new config file we set the path prefix we will need for prod. We may also need to tweak some other settings.